app/viewModels/components/InputTextViewModel.scala

```diff
@@ -42,7 +42,8 @@ object InputTextViewModel {
   ) extends InputTextViewModel
       with AdditionalHtmlViewModel
 
-  case class AddressTextInput(
+  case class TextInput(
```

Comment:
I can see why this name has been changed, but for me this is a little vague. Essentially what we're saying is "use this input type when you have more than one of them" (i.e. on an address form or in this case a coordinates form). Maybe something like `MultiTextInput`?